src/graphics/render_window.rs

```diff
@@ -408,6 +778,19 @@ impl RenderWindow {
     ///
     /// At any given time, only one window may have the input focus to receive input events
     /// such as keystrokes or most mouse events.
+    ///
+    /// # Usage Example
+    ///
+    /// ```no_run
+    /// # use sfml::window::{Event, Style};
+    /// # use sfml::graphics::RenderWindow;
+    /// # // Create a new window with SFML window as name
+    /// # let mut window = RenderWindow::new((800, 600),
+    /// #                              "SFML window",
+    /// #                              Style::CLOSE,
+    /// #                              &Default::default());
+    /// assert_eq!(window.has_focus(), false);
```

Comment:
This is wrong, like the example above. A window is not guaranteed to not have focus when it's created.
Remove this assertion.

Reply:
This is also another small self-explanatory function. I got rid of the usage example for this one.